aws/ebs_test.go

```diff
@@ -108,7 +108,7 @@ func TestNukeEBSVolumes(t *testing.T) {
 	}
 
 	uniqueTestID := "aws-nuke-test-" + util.UniqueID()
-	createTestEC2Instance(t, session, uniqueTestID, false)
+	createTestEBSVolume(t, session, uniqueTestID)
```

Comment:
Why was the test passing if it wasn't doing the right thing? 

One of the most important ideas from TDD is that you should have your tests fail *before* you implement the feature. This ensures the tests are actually checking what you think they are and have clear errors when they fail.

Reply:
Definitely a lapse of judgement on my part here. It never actually created the EBS volume so when it called nuke and checked that the volume doesn't exist (which it won't) the tests pass. Will update this test shortly to be a lot more solid and will also do the same for other tests.